buildSrc/src/main/kotlin/kotlinx.fuzz.src-module.gradle.kts

```diff
@@ -27,6 +27,17 @@ tasks.getByName<KotlinCompile>("compileKotlin") {
 }
 
 tasks.test {
+    val property = System.getProperty("kotlinx.fuzz.logging.level")
+    systemProperties["kotlinx.fuzz.logging.level"] = if (property in LogLevel.values().map { it.name }) {
```

Comment:
I think it will be good if we make this check case insensitive

Reply:
Fixed